src/utils/interfaces.ts

```diff
@@ -0,0 +1,22 @@
+export interface FormConfig {
+  name: string;
+  type: string;
+  className: string;
+  value: string;
+  placeholder: string;
+  validation: object;
+}
+
+export interface CreateUserData {
+  firstName: string;
+  lastName: string;
+  displayName: string;
+  email: string;
+  password: string;
+  phone: string;
+}
+
+export interface LoginUserData {
```

Comment:
LoginUserData o sa fie folosit doar pentru login, putem sa-l pastram in componenta login.

Reply:
Done!